### PR TITLE
[DOCS-355, -216] Collaborate on Pentests

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Collaborate on Pentests | https://deploy-preview-277--cobalt-docs.netlify.app/platform-deep-dive/collaboration/collaborate-on-pentests/ | |

Not sure if that's the right wording for our beta features.

Screenshot 2023-02-06 at 11 37 04

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Pentest Expectations | https://deploy-preview-277--cobalt-docs.netlify.app/getting-started/what-to-expect/ | |
| User Roles and Permissions |https://deploy-preview-277--cobalt-docs.netlify.app/platform-deep-dive/collaboration/organization/user-roles/ | Added one more permission: `Collaborate on pentests in the Cobalt app and in Slack`|

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Collaboration/collaborate-on-pentests.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+As our pentesters share vulnerabilities that they find in real time, you can start [remediating findings](/platform-deep-dive/pentests/findings/remediate-findings/) before the pentest is complete. Review and analyze each [finding](/platform-deep-dive/pentests/findings/). You can:
+
+{{% process-findings %}}
+
+## Use Slack for Communication
+
+In addition to [messaging in the Cobalt app](#collaborate-in-the-cobalt-app), you can communicate with pentesters in a Slack channel dedicated for your pentest. To learn more about Slack channels, read the [Slack documentation](https://slack.com/intl/en-gb/help/articles/360017938993-What-is-a-channel).
+
+{{< alert title="Note" color="primary" >}}
+The Slack channel is available until your pentest is [Closed](/platform-deep-dive/pentests/pentest-process/pentest-states/). Use [messaging in the Cobalt app](#collaborate-in-the-cobalt-app) to collaborate on pentests at any time throughout the pentest, regardless of its [state](/platform-deep-dive/pentests/pentest-process/pentest-states/).
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Collaboration/collaborate-on-pentests.md:57]: @1roshan Do we also send Slack messages to the Pentester Updates tab?

File: content/en/Platform Deep Dive/Collaboration/collaborate-on-pentests.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Collaborate on Pentests"
+linkTitle: "Collaborate on Pentests"
+weight: 10
+description: >
+  Work with pentesters and your team.
+---
+
+{{% pageinfo %}}
+<span style="color:#8348D3">**BETA FEATURE**</span><br>
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Collaboration/collaborate-on-pentests.md:10]: @1roshan We can potentially add something more about this feature being in beta.

**Comment by 1roshan** [comment]
@ana-dashuk-cobalt recorded a 6 minute Loom going through the page.
https://www.loom.com/share/4b254349f686468b93f93c3b69e4529e

**Comment by ana-dashuk-cobalt** [comment]
Hi @1roshan, I addressed your feedback. If the changes look good, please approve the PR.

**Review by 1roshan — APPROVED** [review]
LGTM. even the beta section

**Pull request merged**